### ISSUE TITLE
Adjust 'Validation of prioritised therapeutic targets' figure

### ISSUE THREAD

**Issue by bschilder** [neurogenomics/rare_disease_celltyping#88]
# Provide examples of successful gene therapies that we recapitulated
- [x] Use RPE65 gene therapy in retinal cells as example. Check if it's one of our prioritised hits, and how specific it to the retinal pigment cells.
- [x] Some successful gene therapies are delivered to liver, but precise cell types not necessarily known. Our results can be used to clarify those cell types.

- [x] Adjust Results text to make it clear we can't say much about depleted failed gene therapy targets bc we only have n=1.
- [x] Adjust Results text to specify what the background gene set is for the hypergeometric tests.
- [ ] Include baseline of genes expected by chance.

<img width="650" alt="Screenshot 2024-10-01 at 11 19 24" src="https://github.com/user-attachments/assets/708bdf84-b01d-4c7b-8d2b-20c7c1651713">

**Comment by bschilder** [comment]
> # Provide examples of successful gene therapies that we recapitulated
> * [x]  Use RPE65 gene therapy in retinal cells as example. Check if it's one of our prioritised hits, and how specific it to the retinal pigment cells.

New paragraph in the "Therapeutic target validation" section of the Results.

> **In addition to aggregate enrichment results, we also provide specific examples of successful gene therapies whose cell type-specific mechanism were recapitulated by our phenotype-cell associations. In particular, our pipeline nominated the gene RPE65 within 9 vision-related phenotypes that are hallmarks of 'Leber congenital amaurosis, type II' and 'Severe early-childhood-onset retinal dystrophy'. Indeed, gene therapies targeting RPE65  within the retina of patients with these rare genetic conditions are some of the most successful clinical applications of this technology to date, able to restore vision in many cases [@chiu2021].**

> * [x]  Some successful gene therapies are delivered to liver, but precise cell types not necessarily known. Our results can be used to clarify those cell types.

Right after the previous section: 

> **In other cases, a tissue (e.g. liver) may be known to be causally involved in disease genesis, but the precise causal cell types within that tissue remain unknown (e.g. heptocytes, Kupffer cells, Cholangiocytes, Hepatic stellate cells, Natural killer cells, etc.). Tissue-level investigations (e.g. using bulk transcriptomics or epigenomics) would be dominated by hepatocytes, which comprise 75% of the liver. Our prioritized gene therapy targets can aid in such scenarios by providing the cell type-resolution context most likely to be causal for a given phenotype or set of phenotypes.**

> * [x]  Adjust Results text to make it clear we can't say much about depleted failed gene therapy targets bc we only have n=1.

Caption now reads:
> **Validation of prioritised therapeutic targets. The proportion of existing gene therapy targets (documented in the Therapeutic Target Database) recapitulated by our prioritisation pipeline. Therapetics are stratified by the stage of clinical development they were at during the time of writing. While our prioritized targets did not include any failed ('Terminated') therapies, the fact that only one such therapy exists in the datset preclude us from making any conclusions about depletion of failed gene therapy targets in our prioritised targets list.**

**Comment by bschilder** [comment]
> * [x]  Adjust Results text to specify what the background gene set is for the hypergeometric tests.

Confirmed and added clarification to Results, as well as Methods:
> **The background gene set is composed of the union of all phenotype-associated genes in the HPO and all gene therapy targets listed in the Therapeutic Target Database.**

**Comment by bschilder** [comment]
Updated figure:

![image](https://github.com/user-attachments/assets/b6f30791-8b22-45a3-96b1-3d690da9ff55)


Next step, I need to what "baseline" is and how I should display it.

**Comment by bschilder** [comment]
> - [x] Fix percent labels on top of bars (some are showing 0% by default)

 Fixed:
![image](https://github.com/user-attachments/assets/641c4fcf-8762-4e4d-975a-b193b4f90e23)

**Comment by bschilder** [comment]
> Next step, I need to figure out what "baseline" is and how I should display it.

- So I think baseline should be the proportion of HPO genes (regardless of our prioritisation pipeline) expected to overlap with the TTD targets by chance.
- I computed these baseline proportions within each Approval Stage (x-axis).
- Because our scale is now the number of counts (instead of proportions) I transformed the baseline into the number of targets expected by chance within that Approval Stage.

![image](https://github.com/user-attachments/assets/5b23a369-b8cd-47fa-ab2e-348a2389b536)
